data/wof_snapshot.py

```diff
@@ -11,7 +11,9 @@
 import requests
 from tqdm import tqdm
 
-
+"""
+expects s to look like "123456.geojson"
```

Comment:
nit: extra `s` here

Reply:
that's the name of the var.  I suppose it is unnecessary